packages/@aws-cdk/aws-iot-actions/lib/firehose-stream-action.ts

```diff
@@ -0,0 +1,85 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as iot from '@aws-cdk/aws-iot';
+import * as firehose from '@aws-cdk/aws-kinesisfirehose';
+import { CommonActionProps } from './common-action-props';
+import { singletonActionRole } from './private/role';
+
+/**
+ * Record Separator to be used to separate records.
+ */
+export enum FirehoseStreamRecordSeparator {
+  /**
+   * Separate by a new line
+   */
+  NEWLINE = '\n',
+  /**
+   * Separate by a tab
+   */
+  TAB = '\t',
+  /**
+   * Separate by a windows new line
+   */
+  WINDOWS_NEWLINE = '\r\n',
+  /**
+   * Separate by a commma
+   */
+  COMMA = ',',
```

Comment:
Spaces between please:

```suggestion
  /**
   * Separate by a new line
   */
  NEWLINE = '\n',

  /**
   * Separate by a tab
   */
  TAB = '\t',

  /**
   * Separate by a windows new line
   */
  WINDOWS_NEWLINE = '\r\n',

  /**
   * Separate by a commma
   */
  COMMA = ',',
```